src/codeu/chat/client/commandline/Chat.java

```diff
@@ -186,6 +188,23 @@ private UserContext findUser(String name) {
         return null;
       }
     });
+    
+    //Added this code to respond to the user's request of info
+    panel.register("info", new Panel.Command() {
+  	  @Override
+  	  public void invoke(Scanner args) {
+  	    final ServerInfo info = context.getInfo();
+  	    if (info == null) {
+  	      // Communicate error to user - the server did not send us a valid
+  	              // info object.
```

Comment:
Extremely minor fix, maybe just have the comment on one line for clarity if possible? Actual code looks great.